examples/rbd/block-pod-rox.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: apps/v1
```

Comment:
change the file name from pod to deployment

Reply:
OK